packages/wdio-browserstack-service/src/util.ts

```diff
@@ -622,7 +622,15 @@ export function getUniqueIdentifierForCucumber(world: ITestCaseHookParameter): s
 }
 
 export function getCloudProvider(browser: Browser<'async'> | MultiRemoteBrowser<'async'>): string {
-    if (browser.options && browser.options.hostname && browser.options.hostname.includes('browserstack')) {
+    if (browser && 'instances' in browser) {
```

Comment:
let's use the `instance of` to check for type of Browser to be equal to MultiRemote

Reply:
Using `instanceof MultiRemoteBrowser` is giving compile error with typescript. We need to change the way we import MultiRemoteBrowser from `type` import to `regular` import
```
'MultiRemoteBrowser' cannot be used as a value because it was imported using 'import type'.ts(1361)
```